pkgs/applications/audio/audacity/default.nix

```diff
@@ -74,41 +81,31 @@ let
   wxmac' = wxmac.overrideAttrs (oldAttrs: rec {
     src = wxWidgets_src;
   });
-
 in
 stdenv.mkDerivation rec {
-  pname = "audacity";
-  # nixpkgs-update: no auto update
-  # Humans too! Let's wait to see how the situation with
-  # https://github.com/audacity/audacity/issues/1213 develops before
-  # pulling any updates that are subject to this privacy policy. We
-  # may wish to switch to a fork, but at the time of writing
-  # (2021-07-05) it's too early to tell how well any of the forks will
-  # be maintained.
```

Comment:
https://github.com/audacity/audacity/issues/1213#issuecomment-892546601

> for folks who build Audacity (e.g. distros) networking has to be positively enabled

So in this scenario we've not positively enabled it here have we?

Reply:
@06kellyjac https://github.com/audacity/audacity/blob/ff6d6739fbe73de9c9ad3aa4e07642653604af44/CMakeLists.txt#L173, https://github.com/audacity/audacity/blob/ff6d6739fbe73de9c9ad3aa4e07642653604af44/CMakeLists.txt#L200, and further networking options seem to disable by default with the value Off.